docs/getting_started/architecture.md

```diff
@@ -0,0 +1,77 @@
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import ThemedImage from '@theme/ThemedImage';
+
+# Architecture
+
+Jellyfish ships with a couple of SDKs that make it easier to manage its state, as well as
+publish and consume tracks.
+
+There are two types of SDKs: server side and client side.
+
+## Server SDKs
+
+Server SDKs allow you to manage Jellyfish state.
+In particular, you can:
+* authenticate
+* create/delete rooms
+* add/remove peers
+* add/remove components
+
+Support for server-side notifications is also planned for the near future.
+
+Under the hood, Server SDKs just use [Jellyfish REST API](../api_reference.md). 
+
+:::tip
+
+As an example, you can use server SDK to introduce a time limit on meeting duration and shut down the room after it passes.
+
+:::
+
+## Client SDKs
+
+Client SDKs allow you to connect to Jellyfish to publish and consume tracks.
+In particular, you can:
+* authenticate
+* join specific room
+* publish audio and video
+* receive audio and video from other peers and components
+
+At the moment, Jellyfish supports only one type of client - WebRTC.
+
+## Diagram
+
+The whole concept is presented in the following diagram.
+
+
+<ThemedImage
+  alt="Jellyfish architecture"
+  sources={{
+    light: useBaseUrl('/img/architecture-light.png'),
+    dark: useBaseUrl('/img/architecture-dark.png'),
+  }}
+/>
```

Comment:
This schema is a bit complicated and has lots of elements.
I figured it would be nice to have zoom option to take a closer look at such images.
Take a look at this PR: https://github.com/jellyfish-dev/jellyfish-docs/pull/4